packages/build/src/error/parse/parse.js

```diff
@@ -31,10 +32,24 @@ const getFullErrorInfo = function ({ error, colors, debug }) {
   const { message: messageA, stack: stackA } = getStackInfo({ message, stack, stackType, rawStack, severity, debug })
 
   const pluginInfo = getPluginInfo(plugin, location)
+  const tsConfigInfo = getTsConfigInfo(tsConfig)
   const locationInfo = getLocationInfo({ stack: stackA, location, locationType })
   const errorPropsA = getErrorProps({ errorProps, showErrorProps, colors })
 
-  return { ...basicErrorInfo, title: titleA, message: messageA, pluginInfo, locationInfo, errorProps: errorPropsA }
+  return {
+    ...basicErrorInfo,
+    title: titleA,
+    message: messageA,
+    tsConfigInfo,
+    pluginInfo,
+    locationInfo,
+    errorProps: errorPropsA,
+  }
+}
+
+// Serialize the `tsConfig` error information
+const getTsConfigInfo = function (tsConfig) {
+  return tsConfig === undefined ? tsConfig : serializeObject(tsConfig)
```

Comment:
This is a tiny observation that completely boils down to personal preference, so please feel free to dismiss.

IMO the ternary operator, and the fact that we're returning `tsConfig` when we know for a fact it will be `undefined`, adds a small but unnecessary cognitive effort. Even though it's more verbose, I would personally make an early void return to be more explicit.

```suggestion
  if (tsConfig === undefined) {
    return
  }

  return serializeObject(tsConfig)
```

Reply:
Fixed :+1: